tutorials/financial-tick-data/index.md

```diff
@@ -38,10 +38,10 @@ This tutorial covers:
     [Twelve Data][twelve-data] into your TimescaleDB database.
 1.  [Query your dataset][financial-tick-query]: Create candlestick views, query
     the aggregated data, and visualize the data in Grafana.
-1.  [Ingest real-time websocket data][advanced-websocket]: Ingest real-time
-    time-series data into TimescaleDB database using a websocket connection.
-    Create candlestick views, query the aggregated data, and visualize the data
-    in Grafana.
+    
+    To ingest real-time time-series data into TimescaleDB databaseusing a websocket
+    connection. Create candlestick views, query the aggregated data,and visualize
+    the data in Grafana,see [Ingest real-time websocket data][advanced-websocket].
```

Comment:
I think this requires structural changes too, though 🤔

Reply:
![image](https://github.com/timescale/docs/assets/3914967/0495ad36-4746-4ef9-9c1a-decacfd74913)

Reply:
"Ingest real-time financial websocket data" should be at the same level as "Store financial tick data".

Don't worry about the verticals for now, I'll redo this organisation later on for that.